addons/t9n/views/t9n_menus_views.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="t9n_project_menu" name="Projects" action="t9n_project_action" parent="t9n.menu_root"/>
```

Comment:
```suggestion
    <menuitem id="t9n.project_menu" name="Projects" action="t9n.project_action" parent="t9n.menu_root"/>
```
The "t9n" prefix shouldn't be part of the name, it's a separate prefix that is needed to target the record of a specific module. In your case, it's not even necessary, I'd just put it there to make it easier to grep all occurrences of that particular record in all modules. But in the end, I don't really have a strong opinion on the matter.